cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/testevents/TestEventMessagesEntity.java

```diff
@@ -64,8 +66,12 @@ public TestEventMessagesEntity(StoredTestEventId eventId, Set<StoredMessageId> m
 		logger.debug("Creating entity with event-messages link");
 		
 		this.eventId = eventId.toString();
-		this.messageIds = messageIds.stream().map((id) -> id.toString()).collect(Collectors.toCollection(() -> new LinkedHashSet<>()));
+		this.messageIds = messageIds.stream().map(StoredMessageId::toString).collect(Collectors.toCollection(LinkedHashSet::new));
 		this.instanceId = instanceId;
+		// Amazon Keyspaces does not support frozen data type. 
```

Comment:
if this is for Amazon, why this is done for Cassandra? Maybe, use different comment, e.g. "Caassandra implementation may not support frozen data types..." and the rest of your comment.
Also, this change will require migration of old databases due to new column in key. I think it is better to avoid this, i.e. to use this configuration for Amazon only.